src/OrbitQt/orbitsamplingreport.cpp

```diff
@@ -93,7 +94,38 @@ void OrbitSamplingReport::Initialize(orbit_data_views::DataView* callstack_data_
     //  is no need for manual updates.
     orbit_data_views_.push_back(treeView);
 
-    QString thread_name = QString::fromStdString(report_data_view.GetName());
+    std::string thread_name_std = report_data_view.GetName();
+    QString thread_name = QString::fromStdString(thread_name_std);
+
+    uint32_t thread_id = report_data_view.GetThreadID();
+    // Report any thread that contains more than 5% unwinding errors.
+    constexpr double kUnwindErrorThreshold = 0.05;
+    double unwind_error_ratio = sampling_report_->UnwindErrorRatio(thread_id);
+    if (sampling_report_->UnwindErrorRatio(thread_id) > kUnwindErrorThreshold) {
```

Comment:
Doesn't matter, but if the threshold is %5, I would expect a `>=` here.

Reply:
Yeah, doesn't matter, but done.